components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger/softwareversionchecktrigger/softwareversionchecktrigger_test.go

```diff
@@ -62,7 +62,7 @@ func TestSoftwareVersionCheck_Run(t *testing.T) {
 		require.Len(t, ctr, 2)
 		require.Nil(t, ctr[0].Error)
 		require.Len(t, ctr[0].Result.Checks, 2)
-		require.Equal(t, "API check", ctr[0].Result.Check)
+		require.Equal(t, "", ctr[0].Result.Check)
```

Comment:
please check the logic, I don't think this change is needed.

Reply:
Modified this to previous state